### PR TITLE
PAPP-25071: Fidelis Network initial commit

### PR DESCRIPTION
**Pull request by mmomin-crest**
Only the Integration job is failing because the link is not updated with the latest run.
Manual run pipeline - https://jenkins.engineering.phantom.us:8443/job/run-app-integration-tests/7446/console

#### Please check the type of change your PR introduces:
- [x] New App

### CODE REVIEW

**Review by pdesai-crest — APPROVED** [review]

**Review by pdesai-crest — CHANGES_REQUESTED** [review]
Fix pipeline issues

**Review by pdesai-crest — APPROVED** [review]

**Review by tonyc-phantom — APPROVED** [review]

File: fidelisnetwork.json [hunk 1]
```diff
@@ -0,0 +1,1073 @@
...
+                    "data_path": "action_result.summary.total_alerts",
+                    "example_values": [
+                        512
+                    ],
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.data.*.aaData.*.ALERT_ID",
+                    "example_values": [
+                        false
```
Comment (paulnawat) [inline fidelisnetwork.json]: Should the example value here be `ID`?

File: fidelisnetwork.json [hunk 2]
```diff
@@ -0,0 +1,1073 @@
...
+                    "column_order": 0,
+                    "data_type": "string",
+                    "contains": [
+                        "alert id"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.aaData.*.SEVERITY",
+                    "example_values": [
+                        false
```
Comment (paulnawat) [inline fidelisnetwork.json]: Should the example value here be SEVERITY word? 
I'm surprised to see Boolean value here. The same apply to the `false` value below:

File: fidelisnetwork.json [hunk 3]
```diff
@@ -0,0 +1,1073 @@
...
+                    ],
+                    "column_name": "ALERT TIME",
+                    "column_order": 4,
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.aaData.*.SUMMARY",
+                    "example_values": [
+                        false
```
Comment (paulnawat) [inline fidelisnetwork.json]: Should the value here be some texts instead of boolean?

File: fidelisnetwork.json [hunk 4]
```diff
@@ -0,0 +1,1073 @@
...
+                    "data_type": "string",
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.alertId",
+                    "example_values": [
+                        4086
```
Comment (paulnawat) [inline fidelisnetwork.json:333]: I believe this id value is the same one displayed as part of the `list alerts` above, right? 
The same goes for other fields below here.

File: fidelisnetwork.json [hunk 5]
```diff
@@ -0,0 +1,1073 @@
...
+                {
+                    "data_path": "action_result.data.*.assets.asset.coverageScore",
+                    "example_values": [
+                        2
+                    ],
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.data.*.assets.asset.decryptEnd",
+                    "data_type": "string"
```
Comment (paulnawat) [inline fidelisnetwork.json:431]: Do we have some example values for these fields as well?

File: fidelisnetwork.json [hunk 6]
```diff
@@ -0,0 +1,1073 @@
...
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.parameter.alert_id",
+                    "data_type": "string",
```
Comment (paulnawat) [inline fidelisnetwork.json:1036]: Can we include some example values here?

**Review by paulnawat — CHANGES_REQUESTED** [review]

File: fidelisnetwork.json (hunk 1 above)
Comment (mmomin-crest) [reply fidelisnetwork.json]: Yes, The example value should be ID. I updated it.

File: fidelisnetwork.json (hunk 2 above)
Comment (mmomin-crest) [reply fidelisnetwork.json]: Yes, The example value should be value of severity. I updated it.

File: fidelisnetwork.json (hunk 3 above)
Comment (mmomin-crest) [reply fidelisnetwork.json]: Yes, The example value should be summary of alert. I updated it.

File: fidelisnetwork.json (hunk 4 above)
Comment (mmomin-crest) [reply fidelisnetwork.json:333]: Yes, The id is same one displayed as part of the list alerts above. I updated it.

File: fidelisnetwork.json (hunk 6 above)
Comment (mmomin-crest) [reply fidelisnetwork.json:1036]: Yes, I added example value.

File: fidelisnetwork.json (hunk 5 above)
Comment (mmomin-crest) [reply fidelisnetwork.json:431]: No, We haven't get any values for decryptEnd. We only receiving `null` value for it. Hence, I didn't add example value.

File: fidelisnetwork.json [hunk 7]
```diff
@@ -0,0 +1,1082 @@
...
+                    "data_path": "action_result.data.*.cancelled",
+                    "example_values": [
+                        false
+                    ],
+                    "data_type": "boolean"
+                },
+                {
+                    "data_path": "action_result.data.*.totalUnknown",
+                    "example_values": [
+                        false
```
Comment (paulnawat) [inline fidelisnetwork.json:223]: Should this value be a number?

File: fidelisnetwork.json [hunk 8]
```diff
@@ -0,0 +1,1082 @@
...
+                    ],
+                    "column_name": "SEVERITY",
+                    "column_order": 1,
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.time",
+                    "example_values": [
+                        "Medium"
+                    ],
```
Comment (paulnawat) [inline fidelisnetwork.json]: this value should be timestamp, right?

**Review by paulnawat — APPROVED** [review]
Changes look great. I left just a few more minor comments on the `example values`

File: fidelisnetwork.json (hunk 8 above)
Comment (mmomin-crest) [reply fidelisnetwork.json]: Yes, This should be a timestamp value. I have updated it.

File: fidelisnetwork.json (hunk 7 above)
Comment (mmomin-crest) [reply fidelisnetwork.json:223]: No, We are getting boolean type output response for totalUnknown.
image

**Pull request merged**